dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -248,9 +248,9 @@
     <Thickness x:Key="NavigationViewItemSeparatorMargin">16,10</Thickness>
     <Thickness x:Key="NavigationViewCompactItemSeparatorMargin">16,10</Thickness>
     <Thickness x:Key="TopNavigationViewItemSeparatorMargin">10,0</Thickness>
-    <Thickness x:Key="NavigationViewItemContentPresenterMargin">0,0,16,0</Thickness>
+    <Thickness x:Key="NavigationViewItemContentPresenterMargin">0,0,20,0</Thickness>
     <Thickness x:Key="TopNavigationViewItemContentPresenterMargin">8,0,16,0</Thickness>
-    <Thickness x:Key="NavigationViewItemExpandChevronMargin">-16,0,0,0</Thickness>
+    <Thickness x:Key="NavigationViewItemExpandChevronMargin">-20,0,6,0</Thickness>
```

Comment:
Why are those not changed by the same amount of pixels?

Reply:
The goal with this PR is to have a *spacing of 4px between the chevron and the expanded scrollbar*:

- The expanded scrollbar has a default width of 16px
- The chevron's hit area has a width of 40px, with the actual textblock having a width of 12px. As it is centered inside the hit test area, that leaves (40px - 12px) / 2 = 14px on each side of the chevron. In other words, there is currently a space of 14px between the chevron and the right border of the NavigationViewItem. This means that currently, the expanded scrollbar occludes *2px* of the chevron. In order to get the desired spacing of *4px*, I thus add an additional right padding of 6px. The chevron now has a space of 14px + 6px = 20px beween it and the right border.
- The ContentPresenter currently has a right padding of 16px which is just enough to not be occluded by the expanded scrollbar. However, the right padding now differs with the padding for the chevron causing the text and the chevron not to be visually aligned any longer (see [this](https://user-images.githubusercontent.com/1398851/80279353-6d7d5680-86fd-11ea-8760-b010900f943e.png) image). To get the ContentPresenter and chevron visually aligned again, I need to add the difference in padding to its current margin: 20px (chevron padding) - 16px (ContentPresenter pading) = 4px. That gives you the new margin of 16px + 4px = 20px for the ContentPresenter. To keep the spacing between the chevron and the ContentPresenter the same, the left margin of the chevron has to match that updated padding, hence the -20px you are seeing.

Reply:
Okay I see. Thanks.